config.toml.in

```diff
@@ -35,6 +35,17 @@ type = "file"
   # The number of backups to keep.
   # Specify -1 to allow unlimited backups, 0 to not create backups at all.
   #max = 100
+  
+# type = "s3"
```

Comment:
This is done, but for some reason it's not showing the comment as outdated (probably because the change didn't affect L:39)